src/type/form.ts

```diff
@@ -27,6 +27,13 @@ export interface UseFormParam<FormT> {
   lazy?: UseFormLazy
 }
 
+export interface ValidateOptions { fullResult?: boolean }
+export interface UseFormReturnRuleItem {
+  validate(v: any): boolean
+  validate(v: any, validateOptions?: ValidateOptions): { valid: boolean; message: string | null }
```

Comment:
Why does `message` need the `null` type when the configured default string should be used?

Reply:
I guess I hadn't thought it through here, I'll change it to just `string` message.